.circleci/config.yml

```diff
@@ -30,64 +33,90 @@ commands:
           key: android=tools-v1-{{ checksum "scripts/install-android-tools.sh" }}-{{ arch }}
           paths:
             - ~/android-tools
-
-  create-launch-android-emulator:
-    description: "create and launch android emulators"
-    steps:
-      - run:
-          name: create AVD
-          command: |
-            echo "no" | avdmanager --verbose create avd --force \
-              --name "Pixel_3a_API_26" \
-              --package "system-images;android-26;google_apis_playstore;x86"
-
-      - run:
-          name: start AVD
-          command: emulator @Pixel_3a_API_26 -no-window -no-audio
-          background: true
-
-      - run:
-          name: wait for emulator
-          command: |
-            adb wait-for-device shell 'while [[ -z $(getprop dev.bootcomplete) ]]; do sleep 1; done;'
   
   npm-dependencies:
-    description: "installs all global and local npm dependencies"
+    description: "Install all global and local npm dependencies"
     steps:
       - run: npm install -g cordova
       - run: npm install -g github:apache/cordova-paramedic
+      - run: npm install --legacy-peer-deps
+
+  npm-ios-dependencies:
+    description: "Install all ios global and local npm dependencies"
+    steps:
       - run: npm install -g ios-deploy
-      - restore_cache:
-          keys:
-            - v2-dependencies-{{ checksum "package.json" }}
-            - v2-dependencies-
-      - run: npm install
-      - save_cache:
-          paths:
-            - node_modules
-          key: v2-dependencies-{{ checksum "package.json" }}
+      - run: npm install --legacy-peer-deps
     
+  replace-api-key-linux:
+    description: "Replace API_KEY"
+    steps:
+      - run: sed -i s/api_key/$API_KEY/ tests/tests.js
+
   replace-api-key:
-    description: "replace API_KEY"
+    description: "Replace API_KEY"
     steps:
       - run: sed -i .bck s/api_key/$API_KEY/ tests/tests.js
 
+  install-sdkman:
+    description: Install SDKMAN
+    steps:
+      - run:
+          name: Installing SDKMAN
+          command: |
+            curl -s "https://get.sdkman.io?rcupdate=false" | bash
+            echo -e '\nsource "/home/circleci/.sdkman/bin/sdkman-init.sh"' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Setup Java environment
+          command: |
+            sdk env install
+            echo -e '\nexport JAVA_HOME="/home/circleci/.sdkman/candidates/java/current"' >> $BASH_ENV
```

Comment:
I have my own CircleCI orb that installs SDK man (if we wanted to use it) or create our own RevenueCat orbs to resume 🤷‍♂️ 

See 👉 https://github.com/joshdholtz/circleci-orbs

Reply:
Amazing! I created a tiny orb once https://github.com/RevenueCat/sdks-circleci-orb we can move all these to that repo in the future